app/node_modules/xo-notify/index.js

```diff
@@ -12,6 +13,54 @@ export default angular.module('xo-notify', [
   angularNotifyToaster
 ])
   .service('xoNotify', (toaster) => {
+    const Notification = window.Notification ||
```

Comment:
Use `$windows` instead of `windows` ([reference](http://devdocs.io/angular/ng/service/$window)).